File: project_06.cpp
// Number Guessing Game (random number vs user guess)

#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;
int main()
{
    srand(time(0));
    int secretNumber = rand() % 100 + 1;

    int userNumber;
    int attempts = 0;

    while (true)
    {
        cout << "Enter your guess number : ";
        cin >> userNumber;

        attempts++;

        if (secretNumber == userNumber)
        {
            cout << "Correct! You guessed it in " << attempts << " attempts.";
            break;
        }

        else if (secretNumber > userNumber)
        {
            cout << "Too low, try again" << endl;
        }

        else
        {
            cout << "Too high, try again" << endl;
        }
    }
    return 0;
}

File: project_06.py
# Number Guessing Game (random number vs user guess)

import random
secretNumber = random.randint(1,100)
attempts = 0
while True:
    userNumber = int(input("Enter guess number : "))
    attempts += 1
    if secretNumber == userNumber:
        print(f"Correct! Your guessed it in {attempts} attempts.")
        break
    elif secretNumber > userNumber:
        print("Too low, try again!")
    else:
        print("Too high, try again!")
